src/module_1/module_1_meteo_api.py

```diff
@@ -100,7 +95,7 @@ def daily_to_yearly(daily):
     return yearly
 
 
-def plotting(list):
+def plotting(list, city):
     list1 = daily_to_yearly(list)
     x = range(len(list1["time"]))
```

Comment:
x is a generator, that is though to be consumed once. It would be better to have a list or numpy array (e.g. x=np.arange(len(list1["time"])